lighthouse-cli/test/smokehouse/test-definitions/redirects/redirects-config.js

```diff
@@ -18,6 +18,6 @@ module.exports = {
       'redirects',
     ],
     // Use provided throttling method to test usage of correct navStart.
-    throttlingMethod: 'provided',
+    throttlingMethod: /** @type {'provided'} */ ('provided'),
```

Comment:
I have no earthly idea why my other changes would start triggering a failure here, but [they did](https://github.com/GoogleChrome/lighthouse/runs/895613104#step:11:6) might as well fix it

Reply:
weird